### ISSUE TITLE
Hoverable Drop Down Menu Callouts don't work :(

### ISSUE THREAD

**Issue by VerAn1015** [alangrainger/obsidian-share#104]
Hi. Thank you for this plugin. I have this hoverable drop down menu CSS for callouts that I got from discord. Is it possible to make it work when using share note?

In Obsidian sandbox vault
![Obsidian_4TOSSRUBK1](https://github.com/alangrainger/share-note/assets/148541981/4cfa9c25-a006-42bc-bbf5-04bfc86553a2)
[](url)

In Share Note. It only appears as a normal callout. 
![image](https://github.com/alangrainger/share-note/assets/148541981/d240b6be-043d-4a66-84ba-045f475855cc)

Didn't have any problems with other callouts with functionalities like tabbed or scroller callouts. Just this one. Making it possible to work would be highly appreciated. Thanks!

CSS: https://pastebin.com/Lyps1XYa
MD file: https://pastebin.com/cY3rhZS3

**Comment by alangrainger** [comment]
Did you force re-upload your CSS after activating this snippet?

**Comment by VerAn1015** [comment]
Yes I have. And there are no additional plugins involved to make it work. Just pure CSS

**Comment by aiproductguy** [comment]
I got this working a different way.
It requires nested block quotes and MCL Multi-column css as a snippet (Force re-upload of course).
https://efemkay.github.io/obsidian-modular-css-layout/multi-column/

Example Source
![CleanShot 2024-09-16 at 20 55 55@2x](https://github.com/user-attachments/assets/ac442ddb-0272-4b7b-a2b7-ae606501935f)

Share.note page: https://share.note.sx/i5urvww7#4JOumoJkHb37VkxWzjrn2MQIBWLCzB5ihwUOiegSxYY
